compiler/backend/ccgtypes.nim

```diff
@@ -674,8 +616,7 @@ proc getTypeDescAux(m: BModule, origTyp: PType, check: var IntSet; kind: TSymKin
     return
   case t.kind
   of tyRef, tyPtr, tyVar, tyLent:
-    var star = if t.kind in {tyVar} and tfVarIsPtr notin origTyp.flags and
-                    compileToCpp(m): "&" else: "*"
+    var star = "*"
```

Comment:
```suggestion
    let star = "*"
```
The `star[0] = '*'` in line 628 can also be removed.

Reply:
updated